index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <title>Fork me on GitHub CSS Ribbon</title>
-    <link rel="stylesheet" href="https://oss.maxcdn.com/libs/normalize-css/3.0.0/normalize.min.css">
+    <link rel="stylesheet" href="https://necolas.github.io/normalize.css/8.0.1/normalize.css">
```

Comment:
Where did this CDN link from from? The [README](https://github.com/necolas/normalize.css/blob/master/README.md#normalizecss) links to [yarnpkg.com](https://classic.yarnpkg.com/en/package/normalize.css), which lists a few CDN URLs, but this isn't on the list.

Reply:
I get it on the official website @simonwhitaker [here](https://necolas.github.io/normalize.css/). When you right click it and copy the link address.